sites/updates.thunderbird.net/includes/base/base.html

```diff
@@ -0,0 +1,89 @@
+    <meta property="fb:page_id" content="{% block facebook_id %}348169218537151{% endblock %}">
+    <meta name="twitter:card" content="summary">
+    <meta name="twitter:site" content="@{% block twitter_id %}mozthunderbird{% endblock %}">
+    <meta name="twitter:domain" content="thunderbird.net">
+    <link rel="apple-touch-icon" type="image/png" sizes="180x180" href="{% block page_ios_icon %}{{ static('img/thunderbird/ios-icon-180.png') }}{% endblock %}">
+    <link rel="icon" type="image/png" sizes="196x196" href="{% block page_favicon_large %}{{ static('img/thunderbird/favicon-196.png') }}{% endblock %}">
+    <link rel="shortcut icon" href="{% block page_favicon %}{{ static('img/thunderbird/favicon.ico') }}{% endblock %}">
+
+    {% endblock shared_meta %}
+    <link href="{{ static('css/normalize.css') }}" rel="stylesheet" type="text/css" />
+    {% block base_css %}
+    <link href="{{ static('css/updates-style.css') }}" rel="stylesheet" type="text/css" />
+    {% endblock %}
+    {% block site_js %}
+      {#- site-bundle should block html rendering so we can prevent no-js/js element flashing. -#}
+      <script type="text/javascript" src="{{ static('js/common-bundle.js') }}" charset="utf-8"></script>
+    {% endblock %}
+
+    {{ l10n_css() }}
+
+    <!--[if !lte IE 8]><!-->
+    {# Global styles, hidden from IE8 and lower #}
+    {% block site_css %}{% endblock %}
+
+    {# Page-specific styles, hidden from IE8 and lower #}
+    {% block page_css %}{% endblock %}
+    <!--<![endif]-->
+
+    {% block extrahead %}
+      {# Extra header stuff (scripts, styles, metadata, etc) seen by all browsers. Use the 'page_css' block for CSS you want to hide from IE8 and lower. #}
+    {% endblock %}
+    <script>
+      window.siteLocale = "{{ LANG }}";
+    </script>
+  </head>
+
+  <body {% if self.body_id() %}id="{% block body_id %}{% endblock %}" {% endif %}class="html-{{ DIR }} {% block body_class %}{% endblock %}" {% block body_attrs %}{% endblock %}>
+    {# for headers not to be confined by #wrapper (like fx family nav) #}
+    {% block site_header_unwrapped %}{% endblock %}
+
+    {% block site_nav %}{% endblock %}
+
+    {% block site_header %}
+    {% endblock %}
+
+    <main>
+      {% block pre_content %}{% endblock %}
+      <div id="main-content">
+      {% block content %}{% endblock %}
+      </div>
+      {% block post_content %}{% endblock %}
+    </main>
+
+    {% block site_footer %}
+        {% include 'includes/base/footer.html' %}
+    {% endblock %}
+
+    {% block additional_site_js %}{% endblock %}
+    {% block tracker_js %}
+      <script type="text/javascript" src="{{ static('js/matomo-config.js') }}" charset="utf-8"></script>
+      <script type="text/javascript" src="{{ static('js/matomo.js') }}" async defer></script>
+      <noscript>
+        <p><img src="//thunderbird.innocraft.cloud/piwik.php?idsite=1&rec=1" style="border:0;" alt="" /></p>
```

Comment:
@Sancus do we need a new matomo site? Or are we just scoping the metrics?

Reply:
Maybe we should make another site. The alternative is a segment, I guess, but so far the segmentation in Matomo seems to work very poorly if at all.